multiqc/modules/hicexplorer/hicexplorer.py

```diff
@@ -0,0 +1,184 @@
+
+            #### Same fragment
+
+            These are read mates, facing inward, separated by up to 800 bp that do not have a restriction enzyme in between.
+            These read pairs are not valid Hi-C pairs.
+
+            #### Self circle
+
+            Self circles are defined as pairs within 25kb with \'outward\' read orientation
+
+            #### Self ligation
+
+            These are read pairs with a restriction site in between that are within 800 bp.'''
+        )
+
+        self.add_section(
+            name='Contact distance',
+            anchor='hicexplorer_contact_distance',
+            plot=self.hicexplorer_create_plot(keys_list_contact_distance, 'HiCExplorer: Categorization of reads')
+        )
+
+        self.add_section(
+            name='Read orientation',
+            anchor='hicexplorer_read_orientation',
+            plot=self.hicexplorer_create_plot(keys_list_read_orientation, 'HiCExplorer: Read orientation')
+        )
+
+    def parse_logs(self, f):
+        """Parse a given HiCExplorer log file from hicBuildMatrix."""
+        data = {}
+        for l in f.splitlines():
+            # catch empty lines
+            if len(l) == 0:
+                continue
+            s = l.split("\t")
+            data_ = []
+            # catch lines with descriptive content: "Of pairs used:"
+            for i in s[1:]:
+                if len(i) == 0:
+                    continue
+                try:
+                    i.replace('(', '')
+                    i.replace(')', '')
+                    i.replace(',', '')
+
+                    data_.append(float(i))
+                except ValueError:
+                    data_.append(i)
+            if len(data_) == 0:
+                continue
+            if s[0].startswith('short range'):
+                s[0] = 'short range'
+            elif s[0].startswith('same fragment'):
+                s[0] = 'same fragment'
+            s[0] = s[0].capitalize()
+            data[s[0]] = tuple(data_)
+        return data
+
+    def hicexplorer_basic_statistics(self):
+        """Create the general statistics for HiCExplorer."""
+        data = {}
+
+        for file in self.mod_data:
+            total_pairs = self.mod_data[file]['Pairs considered'][0]
+            data_ = {
+                'Pairs considered': self.mod_data[file]['Pairs considered'][0],
+
+                'Pairs used': self.mod_data[file]['Pairs used'][0] / total_pairs,
+                'Mapped': self.mod_data[file]['One mate unmapped'][0] / total_pairs,
+                'Min rest. site distance': self.mod_data[file]['Min rest. site distance'][0],
+                'Max rest. site distance': self.mod_data[file]['Max rest. site distance'][0],
+            }
+            data[self.mod_data[file]['File'][0]] = data_
+        headers = OrderedDict()
+
+        headers['Pairs considered'] = {
+            'title': 'M Number of pairs',
```

Comment:
This is a bit messy currently, but users can configure what multiplier to use for read counts (millions, thousands etc). So this should be as follows to reflect this:

```python
'title': '{} Pairs'.format(config.read_count_prefix),
```

See http://multiqc.info/docs/#number-base-multiplier for more info.